website/data/community.json

```diff
@@ -541,5 +538,30 @@
     "type": "demos",
     "badges": [],
     "description": "The Connectifi FDC3 Sandbox is a free FDC3 development tool that uses the Connectifi cloud service to securely enable interoperability across applications regardless of technology or device they are running on.  It uses the FDC3 standard and works without installs so that FDC3 can be used directly in a browser, bridging across desktop containers and across devices."
+  },
+  {
+    "title": "Morgan Stanley",
+    "publisher": "Morgan Stanley",
+    "image": "/img/users/MorganStanley.png",
+    "infoLink": "https://github.com/morganstanley/ComposeUI",
+    "docsLink": "http://opensource.morganstanley.com/ComposeUI/documentation/",
+    "type": "platform-provider",
```

Comment:
I would think you could probably make two entries - one as 'adopter' with title Morgan Stanley and a second as platform-provider for ComposeUI specifically. Even better if the latter has its own logo...